Dockerfile

```diff
@@ -1,8 +1,7 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi9/go-toolset:1.18.4 as builder
+FROM docker.io/library/golang:1.19 as builder
```

Comment:
is this intentional ? we should find and go1.19 from redhat registry right ?

Reply:
yes it is intentional, ubi's go-toolset often lies behind in terms of go version so we want something more up to date.
This is for the upstream/community build, so using `golang:1.19` is fine (we actually do the same already for a while in flowlogs-pipeline); the Dockerfile for downstream is different and uses images from redhat registry.